client/src/components/ChargesList.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './PaymentContent.css';
+
+/**
+ * Content for page 0.
+ */
+class PaymentContent0 extends Component {
```

Comment:
Let's actually add some better and more descriptive naming to this so we won't be confused in the future.